### PR TITLE
feat[Assignment 1]: 0.1 git basics

### PR DESCRIPTION
**Pull request by AlgorithmChopda**
Solved Assigment 1 
1.  Write a program to calculate the simple interest
2.  Write a program to calculate the area of the circle

### CODE REVIEW

File: Assignment1/simple_interest.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+*/
+
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func roundFloatTillTwoDecimal(interest float64) float64 {
```
Comment (Deepak1124) [inline Assignment1/simple_interest.go:14]: Seems we haven't called this function.

File: Assignment1/simple_interest.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+func calSimpleInterest(principleAmount, rateOfInterest, time float64) float64 {
+	var interest = (principleAmount * rateOfInterest * time) / 100
+	return interest
+}
+
+func main() {
+	var principleAmount, rateOfInterest, time float64
+	fmt.Println("Enter Principle Amount, Rate Of Interest and Time ")
```
Comment (Deepak1124) [inline Assignment1/simple_interest.go]: Add colon in the end

**Review by Deepak1124 — CHANGES_REQUESTED** [review]

File: Assignment1/simple_interest.go (hunk 1 above)
Comment (AlgorithmChopda) [reply Assignment1/simple_interest.go:14]: done

File: Assignment1/simple_interest.go (hunk 2 above)
Comment (AlgorithmChopda) [reply Assignment1/simple_interest.go]: done

**Pull request merged**